tools/distrib/python/grpcio_tools/grpc_tools/protoc.py

```diff
@@ -18,7 +18,7 @@
 import sys
 
 from grpc_tools import _protoc_compiler
-import pkg_resources
+from importlib.resources import files
```

Comment:
Looks like `importlib.resources.files` is [only available for Python 3.9 and later](https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files), since we still need support 3.7 and 3.8, we'll need a replacement for older versions.

Reply:
@XuanWang-Amos I have tried to catch the version and thus given a soft solution. Will you please have a look and guide me if possible?